src/compiler/diagnosticMessages.json

```diff
@@ -1545,7 +1559,30 @@
         "category": "Error",
         "code": 2510
     },
-
+    "Cannot create an instance of the abstract class '{0}'.": {
+        "category": "Error",
+        "code": 2511
+    },
+    "All overload signatures must match with respect to modifier '{0}'.": {

```

Comment:
Or all overload signatures must be not abstract. I think "match w/r/t modifier `abstract`" captures this slightly better. 

On the other hand, that other modifier error messages use your language, so I think for consistency's sake it's better to make the change.

Per another comment, I'll remove the parameter.